config/eslint.browser.js

```diff
@@ -59,5 +59,6 @@ module.exports = {
         groups: [['builtin', 'external'], ['internal', 'parent', 'sibling', 'index']],
       },
     ],
+    'import/no-cycle': 'off',
```

Comment:
We should remove the dependency cycles in a follow-up.
Brief look - this should not be so difficult.